File: app/visual-model-designer/src/main/java/org/visual/model/app/command/OpenCommand.java
package org.visual.model.app.command;

import lombok.extern.slf4j.Slf4j;
import picocli.CommandLine;

@CommandLine.Command(name = "open", aliases = "o")
@Slf4j
public class OpenCommand implements Runnable {
    @Override
    public void run() {
        log.info("open");
    }
}

File: visual-model-app/src/main/kotlin/org/visual/model/app/command/OpenCommand.kt
package org.visual.model.app.command

import org.visual.model.shared.KSlf4j
import org.visual.model.shared.KSlf4j.Companion.log
import picocli.CommandLine
import java.util.concurrent.Callable

@CommandLine.Command(name = "open", aliases = ["o"])
@KSlf4j
class OpenCommand:Callable<Int> {
    override fun call(): Int {
        log.info("open")
        return 0
    }
}